mne/io/fieldtrip/tests/helpers.py

```diff
@@ -0,0 +1,285 @@
+
+pandas_not_found_warning_msg = 'The Pandas library is not installed. Not ' \
+                               'returning the original trialinfo matrix as ' \
+                               'metadata.'
+
+
+def _has_h5py():
+    try:
+        import h5py  # noqa
+        return True
+    except ImportError:
+        return False
+
+
+def _remove_ignored_ch_fields(info):
+    if 'chs' in info:
+        for cur_ch in info['chs']:
+            for cur_field in ch_ignore_fields:
+                if cur_field in cur_ch:
+                    del cur_ch[cur_field]
+
+
+def _remove_tangential_plane_from_ori(info):
+    if 'chs' in info:
+        for cur_ch in info['chs']:
+            cur_ch['loc'][3:9] = 0
+            cur_ch['loc'][3:] = np.around(cur_ch['loc'][3:], 2)
+            cur_ch['loc'] = np.around(cur_ch['loc'], 3)
+
+
+def _remove_long_info_fields(info):
+    for cur_field in info_long_fields:
+        if cur_field in info:
+            del info[cur_field]
+
+
+def _remove_ignored_info_fields(info):
+    for cur_field in info_ignored_fields:
+        if cur_field in info:
+            del info[cur_field]
+
+    _remove_ignored_ch_fields(info)
+
+
+def get_data_paths(system):
+    """Return common paths for all tests."""
+    test_data_folder_ft = os.path.join(mne.datasets.testing.data_path(),
+                                       'fieldtrip/ft_test_data', system)
+
+    return test_data_folder_ft
+
+
+def get_cfg_local(system):
+    """Return cfg_local field for the system."""
+    from mne.externals.pymatreader import read_mat
+    cfg_local = read_mat(os.path.join(get_data_paths(system), 'raw_v7.mat'),
+                         ['cfg_local'])['cfg_local']
+
+    return cfg_local
+
+
+def get_raw_info(system):
+    """Return the info dict of the raw data."""
+    cfg_local = get_cfg_local(system)
+
+    raw_data_file = os.path.join(mne.datasets.testing.data_path(),
+                                 cfg_local['file_name'])
+    reader_function = system_to_reader_fn_dict[system]
+
+    info = reader_function(raw_data_file, preload=False).info
+    info['comps'] = []
+    return info
+
+
+def get_raw_data(system, drop_sti_cnt=True, drop_extra_chs=False):
+    """Find, load and process the raw data."""
+    cfg_local = get_cfg_local(system)
+
+    raw_data_file = os.path.join(mne.datasets.testing.data_path(),
+                                 cfg_local['file_name'])
+    reader_function = system_to_reader_fn_dict[system]
+
+    raw_data = reader_function(raw_data_file, preload=True)
+    crop = min(cfg_local['crop'], np.max(raw_data.times))
+    if system == 'eximia':
+        crop -= 0.5 * (1.0 / raw_data.info['sfreq'])
+    raw_data.crop(0, crop)
+    raw_data.set_eeg_reference([])
+    raw_data.del_proj('all')
+    raw_data.info['comps'] = []
+    if system == 'KIT':
+        dropped_chan_idx = [9, 19, 29]
+        drop_chans = [raw_data.ch_names[idx] for idx in dropped_chan_idx]
+        raw_data.drop_channels(drop_chans)
```

Comment:
yes, i deleted the code. it was not used anyhow because i have not yet come up with a good way to test KIT files because, as stated in the comments and in `io.rst` KIT import does not work because the channel names are different in FieldTrip and MNE Python to the extent that there is no reliable way to recover that information.